front/src/common/Map/Search/useSearchOperationalPoint.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState, useEffect, useMemo } from 'react';
+
+import { compact } from 'lodash';
+
+import { type SearchResultItemOperationalPoint, osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { useInfraID } from 'common/osrdContext';
+import { useDebounce } from 'utils/helpers';
+
+export const mainOperationalPointsCHCodes = ['', '00', 'BV'];
```

Comment:
I would prefer screaming case for static array like this (MAIN_OP_CH_CODES). Also if you don't replace its other usages, you should not export it